package.json

```diff
@@ -34,9 +34,9 @@
     ]
   },
   "scripts": {
-    "start": "webpack-dev-server --inline --hot --content-base dist/",
+    "start": "webpack serve --inline --hot --content-base dist/",
     "build": "npm-run-all lint:build clean build:webpack",
-    "build:webpack": "webpack --mode production --devtool false",
+    "build:webpack": "webpack --mode production",
```

Comment:
`devtool` can be specified in the `webpack.config.js`.

Reply:
I see we specify `devtool: 'source-map',` by default.  But I'm assuming we want to turn this off if we are in production mode?

Reply:
I see CLUE uses `devtool: devMode ? 'cheap-module-eval-source-map' : 'source-map',`  Wasn't what I expected to see so maybe I'm not understanding the arguments.

Reply:
That is a good question. It can be handy to have sourcemaps in production. This table describes different devTool options. I don't know yet what the production column means:
https://webpack.js.org/configuration/devtool/

Reply:
reading that page more, it sounds like we need to explicitly change the devTool setting in production mode, it doesn't seem like it is done automatically. If we were using sourceMaps in production before we might as well continue to not use them.

Reply:
@mklewandowski what you are seeing with CLUE makes some sense. In devMode it is using eval bases source-maps which are recommended at the link above, and separate file source maps in production.  Since our source code is open source anyhow providing production source maps doesn't seem bad.

Reply:
I'm fine with setting it up the CLUE way if others are ok with that.  Thoughts?

Reply:
I think for now we should try it the CLUE way.  I had to tweak it just a little to fit the arg structure:
```
devtool: devMode ? 'eval-cheap-module-source-map' : 'source-map',
```